tests/pyfunc/test_spark.py

```diff
@@ -56,11 +56,7 @@ def configure_environment():
 
 def get_spark_session(conf):
     conf.set(key="spark_session.python.worker.reuse", value=True)
-    return (
-        pyspark.sql.SparkSession.builder.config(conf=conf)
-        .master("local-cluster[2, 1, 1024]")
-        .getOrCreate()
-    )
+    return pyspark.sql.SparkSession.builder.config(conf=conf).master("local[*]").getOrCreate()
```

Comment:
`local-cluser` doesn't seem to work: https://github.com/mlflow/mlflow/runs/2388530643